modules/programs/nushell.nix

```diff
@@ -21,7 +29,27 @@ in {
       description = "The package to use for nushell.";
     };
 
-    settings = mkOption {
+    config = mkOption {
+      type = types.lines;
+      default = "";
+      defaultText = literalExpression ''""'';
```

Comment:
I don't think `defaultText` isn't necessary for default-empty strings (at least I can't find another instance of that in the repo.